pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!--
-  ~ Copyright 2014-2018 the original author or authors.
+  ~ Copyright 2020 the original author or authors.
```

Comment:
Shouldn't we stick with original dates, eg. 2014? I red here https://stackoverflow.com/a/2391555/2970422 that you give away your copyright for previous years like that. 

We can go for 2020 and later on adjust it again to 2014? When we add the licence header in pom.xml

Reply:
I will revert those changes. I am voting for removing them completely, you can see the history via GIT anyways? (out of scope for this PR I guess)

Reply:
On a side note: 2014 seems a bit early :)